README.md

```diff
@@ -66,12 +66,18 @@ Lo primero que debes crear es el objeto de requerimiento para el SDK el cual se
 
 ````javascript 1.5
 var options = {
-  endpoint: './transaction-create',
-  commerceLogo: '/onepay-sdk-example/images/icons/logo-01.png',
-  callbackUrl: './transaction-commit'
+  endpoint: '//tu-url.com/transaction-create',
+  commerceLogo: '//tu-url.com/images/icons/logo-01.png',
+  callbackUrl: '//tu-url.com/transaction-commit'
 };
 ````
 
+Debido a que el modal reside en un dominio fura de tu comercio, debes usar URLs completas para que los callbacks
+funcionen como se espera.
+
+**NOTA:** Evita poner `http:` o `https:` al inicio de tus urls para evitar problemas de llamada entre diferentes
```

Comment:
This can potentially be bad advise (because you can end up not using https in production when you must use it). I would but it the other way around, like:

> **Tip**: En desarrollo puedes comenzar tus URLS con `//` en lugar de `http://` o `https://` para evitar problemas al mezclar páginas seguras con inseguras. Pero en producción siempre debieras usar https.